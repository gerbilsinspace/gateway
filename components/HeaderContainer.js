import React from 'react';
import Header from './Header';
import H1 from './H1';

const HeaderContainer = ({ text, image, colour }) => (
  <Header image={image} colour={colour}>
    <H1>{text}</H1>
  </Header>
);

export default HeaderContainer;
