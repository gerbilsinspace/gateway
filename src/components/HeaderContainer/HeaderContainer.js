import React from 'react';
import Header from '../Header';
import H1 from '../H1';

const HeaderContainer = ({ text, image, colour }) => {
  if (colour ==='red') {
    colour = '#f05a64';
  }
  return (
    <Header image={image} colour={colour}>
      <H1>{text}</H1>
    </Header>
  );
};

export default HeaderContainer;
