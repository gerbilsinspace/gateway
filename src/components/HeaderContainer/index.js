import React from 'react';
import Header from '../Header';
import H1 from '../H1';

const HeaderContainer = ({ text, image }) => (
  <Header>
    <H1>{text}</H1>
    <img src={image} alt='' />
  </Header>
);

export default HeaderContainer;
