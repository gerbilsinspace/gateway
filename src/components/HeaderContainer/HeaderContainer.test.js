import React from 'react';
import { shallow } from 'enzyme';
import HeaderContainer from './index';

let component;

describe('HeaderContainer', () => {
  beforeEach(() => {
    component = shallow(<HeaderContainer />);
  });

  it('should render without crashing', () => {
    expect(component).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
})