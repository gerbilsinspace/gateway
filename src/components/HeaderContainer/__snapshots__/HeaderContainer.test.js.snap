// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContainer should match snapshot 1`] = `
<styled.header>
  <styled.h1 />
  <img
    alt=""
  />
</styled.header>
`;
